cmd/api-linter/main.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Api-lint is an extendable and configurable .proto file linter that allows developers to provide
```

Comment:
Changed it to `The binary api-linter is a linter that checks Google APIs according to the API Improvement Proposals defined in https://aip.dev.`